ethers-providers/src/transports/quorum.rs

```diff
@@ -518,9 +571,6 @@ where
             // TODO: to robustly support eip-1559, we will likely need to also support
             // eth_feeHistory. This returns an object with various numbers rather than a
             // single number, so we'll need some additional code to handle this case.
-
-            // For RPCs that return numbers that can vary amongst inner providers, come to quorum on
```

Comment:
any reason to remove? this still feels accurate

Reply:
oops xd